core/src/components/datetime/datetime.scss

```diff
@@ -423,6 +422,6 @@ ion-picker-column-internal {
 
 // Year Picker
 // -----------------------------------
-:host(.show-month-and-year) .calendar-action-buttons ion-item {
-  --color: #{current-color(base)};
+:host(.show-month-and-year) .calendar-action-buttons .calendar-month-year-button {
```

Comment:
Looks like there still is a `--color` variable on `ion-button`: https://ionicframework.com/docs/api/button#css-custom-properties-1

Should we keep that usage?

Reply:
The `text-inherit` mixin resets the `color` style property to `inherit`. I'll revisit what style inherits are actually needed for this element and then move away from the mixin so we can use the `--color` CSS variable.